src/backend/data/feed.js

```diff
@@ -115,6 +116,14 @@ class Feed {
     const id = urlToId(url);
     return this.byId(id);
   }
+
+  /**
+   * Returns all the feeds
+   * Returns a Promise<Feeds>
+   */
+  static all() {
+    return getFeeds();
```

Comment:
We will need to file a follow-up to fix all our other calls to `getFeeds()` and replace with this.

I think that `Feeds.all()` should be returning Feed objects.  So this should be something like:

```js
static async all() {
  const ids = await getFeeds();
  return Promise.all(ids.map(Feed.byId));
}
```